xml/cap_overview.xml

```diff
@@ -232,10 +232,10 @@
   </itemizedlist>
 
   <para>
-   &scf; uses &operator;, a &kube; Operator deployed via a &helm; chart, to
+   &productname; uses &operator;, a &kube; Operator deployed via a &helm; chart, to
    install custom resource definitions that convert &bosh; releases into
-   &kube; resources, such as pods, deployments, and stateful sets. Leveraging
-   &operator; is &kubecf;, a version of &cf; deployed as &helm; chart.
+   &kube; resources, such as pods, deployments, and stateful sets. This is made possible
```

Comment:
Probably more accurate to refer to them as Pod, Deployment, and StatefulSet in this instance (maybe enclosed with literal tags?). Also, I think statefulset is always one word based on official Kubernetes material I've seen.

Reply:
Good catch, thanks!